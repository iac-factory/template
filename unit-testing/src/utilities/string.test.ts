import { initialize } from ".";

describe( "C-String-Module", () => {
    initialize();

    it( "Export(s)", async () => {
        expect.assertions( 1 );

        const { C } = await import(".");

        const snapshot = C;

        const state: import("Unit-Testing").State = {
            ... expect.getState(), ... {
                data: { C }
            }
        };

        expect.setState( state );

        expect(snapshot).toMatchSnapshot();
    } );
} );
