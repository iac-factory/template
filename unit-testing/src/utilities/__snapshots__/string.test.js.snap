// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`C-String-Module Export(s) 1`] = `
Object {
  "format": [Function],
  "inspect": [Function],
  "printf": [Function],
  "strip": [Function],
}
`;
